### DISCUSSION TITLE
Expected training time

### DISCUSSION THREAD

**Discussion by usoylu2** [k2-fsa/icefall#1090] [Q&A] [answered]
What is the expected training time for a model like zipformer (zipformer + pruned stateless transducer)
here https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/RESULTS.md
by using Amazon EC2 p3x16 instance which has 8 NVIDIA® V100 Tensor Core GPUs
Is it on the order of 1-2 days or 1-2weeks?

**Comment by yfyeung** [comment, marked as answer]
If you use fp16, it takes less than 1 days to train 30 epochs.
